### DISCUSSION TITLE
docs/os-basic/

### DISCUSSION THREAD

**Discussion by giscus** [ricsanfre/pi-cluster#257] [Announcements]
# docs/os-basic/

Basic Ubuntu OS configurations of our Raspberry Pi cluster nodes.

https://picluster.ricsanfre.com/docs/os-basic/

**Comment by bpartin2009** [comment]
linux-modules-extra-raspi package can not be found..... did they remove it from the repository?

**Reply by bpartin2009** [reply]
Nevermind......I'm an idiot

**Comment by the-paulus** [comment]
I noticed some typos:

```bash
sudo apt install libraspberrypi-bi
```

should be:

```bash
sudo apt install libraspberrypi-bin
```

And I believe

```
gpu_mem=16
````

should be

```
gpu_mem=16M
````
